site/_js/firebase.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @fileoverview Firebase integration.
+ */
+import {initializeApp} from 'firebase/app';
```

Comment:
As mentioned, please load the Firebase runtime from a CDN URL, and only include the code that references Firebase in script that runs on `/100/`. We do not want any Firebase-related code being included in script bundles that execute outside of `/100/`.

Reply:
Agreed. Moved to CDN.